src/test/java/com/ncc/aif/TestUtils.java

```diff
@@ -0,0 +1,466 @@
+     *
+     * @param annotationNamespace namespace to use with URIs
+     * @param validator           an AIF validator instantiated based on the caller's ontology and desired NIST restrictions
+     * @param forceDump           whether or not to force dumping of models prior to validation
+     */
+    NistHypothesisTestUtils(String annotationNamespace, ValidateAIF validator, boolean forceDump) {
+        super(annotationNamespace, validator, forceDump);
+    }
+
+    /**
+     * Makes and returns a valid NIST-restricted entity of the specified type and its cluster with the specified
+     * cluster handle.
+     *
+     * @param type          entity type
+     * @param clusterHandle cluster handle for the entity cluster
+     * @return a key-value Pair of the entity Resource (key) and its associated cluster Resource (value)
+     */
+    Pair<Resource, Resource> makeNistEntity(Resource type, String clusterHandle) {
+        Pair<Resource, Resource> pair = makeNistEntity(type);
+        pair.getValue().addProperty(AidaAnnotationOntology.HANDLE, clusterHandle);
+        return pair;
+    }
+
+    /**
+     * Makes and returns a valid NIST-restricted event of the specified type and its cluster marked with the
+     * specified importance.
+     *
+     * @param type       event type
+     * @param importance the importance to mark the event cluster
+     * @return a key-value Pair of the event Resource (key) and its associated cluster Resource (value)
+     */
+    Pair<Resource, Resource> makeNistEvent(Resource type, double importance) {
+        Pair<Resource, Resource> pair = makeNistEvent(type);
+        markImportance(pair.getValue(), importance);
+        return pair;
+    }
+
+    /**
+     * Makes and returns a valid NIST-restricted relation of the specified type and its cluster marked with the
+     * specified importance.
+     *
+     * @param type       relation type
+     * @param importance the importance to mark the relation cluster
+     * @return a key-value Pair of the event Resource (key) and its associated cluster Resource (value)
+     */
+    Pair<Resource, Resource> makeNistRelation(Resource type, double importance) {
+        Pair<Resource, Resource> pair = makeNistRelation(type);
+        markImportance(pair.getValue(), importance);
+        return pair;
+    }
+
+    /**
+     * Makes and returns an edge relationship between the specified event or relation and an argument filler entity.
+     *
+     * @param eventOrRelation The event or relation for which to mark the specified argument role
+     * @param type            the type of the argument
+     * @param argumentFiller  the filler (object) of the argument
+     * @param importance      the importance to mark the edge
+     * @return the created event or relation argument assertion
+     */
+    Resource makeEdge(Resource eventOrRelation, Resource type, Resource argumentFiller, double importance) {
```

Comment:
Based on the main method comment (compared to the other methods), it looks like this is the only method that can make invalid AIF.  Can you confirm?  All the more important to put the word "Valid" in the other method names.

Reply:
I'm not following.  How is this method suggesting that it can make invalid AIF (other that GIGO)?  As far as I know, it should make valid AIF as long as the parameters are valid (e.g., not sending a mismatched type).

Reply:
The method comment of makeNistEvent says "Makes and returns a valid NIST-restricted event of the specified type and its cluster."  All the other method comments similarly say they make valid this or valid that.  Not so for makeEdge.  It just says "Makes and returns an edge relationship between the specified event or relation and an argument filler entity."  I was thinking that leaving the word "valid" out of the main method comment was intentional and did some spot checking on the use of it (e.g., it's use in invalidRelationAndEventEdge()).

So, I could definitely be wrong, but that was my thinking on my comment on this line.

Reply:
Hmm, the power of copy and paste, I guess.  `makeEdge()` is sort of an outlier in that it doesn't return an entity, event, relation, or hypothesis.  I probably wrote its JavaDoc solo instead of copying it and modifying it from another.

How about if I update the JavaDoc to say:
> Makes and returns a valid argument assertion between the specified event or relation and an argument filler entity.